src/InstallDir/InstallDirDialog.jsx

```diff
@@ -64,3 +64,50 @@ export const selectInstallDir = () => async dispatch => {
         dispatch(hideInstallDirDialog());
     }
 };
+
+const InstallDirDialog = ({ justConfirm }) => {
+    const dispatch = useDispatch();
+    const isVisible = useSelector(isDialogVisible);
+    const { environmentVersionToInstall } = useSelector(state => state.app.environments);
+
+    const confirmDirDialogProps = {
+        title: 'Confirm installation directory',
+        onConfirm: () => selectInstallDir(dispatch),
+        onCancel: () => dispatch(hideInstallDirDialog()),
+    };
+    const changeDirDialogProps = {
+        title: 'Change install directory',
+        confirmLabel: 'Continue installation',
+        optionalLabel: 'Change directory',
+        onConfirm: () => dispatch(installLatestToolchain(environmentVersionToInstall)),
+        onCancel: () => dispatch(hideInstallDirDialog()),
+        onOptional: () => dispatch(selectInstallDir()),
+    };
+    const dialogProps = justConfirm ? confirmDirDialogProps : changeDirDialogProps;
+
+    return (
+        <ConfirmationDialog
+            isVisible={isVisible}
+            {...dialogProps}
+        >
+            <p>
+                <code>{installDir()}</code> is your current installation base directory.
+            Any new installation will be a subdirectory here.
+            </p>
+            <p>
+            When you change the installation directory, SDK environments installed in the old
+            directory will not be shown in the list anymore. They will not be deleted, so
+            you can still find them on the disc and changing back to the old directory will
+            show them in the manager again.
```

Comment:
I mostly kept these texts. We could improve the text, but we also hope that we improve the toolchain manager after the MVP so that changing the installation directory does not make old environments disappear. This enhancement would make this text obsolete anyhow.